e2e-tests/common/utils.resource

```diff
@@ -7,111 +7,92 @@ Library             Process
 
 
 *** Variables ***
-${IMAGES_DIR}    ${CURDIR}/images
 ${SCRIPTS_DIR}    ${CURDIR}/scripts
 
 
 *** Keywords ***
 Log In
-    Match Text    Not listed    120
+    Match Text    Not listed    300
     Hid.Keys Combo    Return
     BuiltIn.Sleep    2
     Hid.Type String    ubuntu
     BuiltIn.Sleep    2
     Hid.Keys Combo    Return
     BuiltIn.Sleep    2
-    Move Pointer To Proportional (0, 1)
-    Match Text    Show Apps    15
+    # Give some time for the desktop to settle
+    BuiltIn.Sleep    5
+    Hid.Move Pointer To Proportional    0    1
+    Match Text    Show Apps    300
 
 
 Log Out
     Run Command    gnome-session-quit --logout --no-prompt
 
 
-Open GNOME Terminal
-    Hid.Keys Combo    Control_L    Alt_L    t
-    Match Text    @ubuntu:~$    15
+Open Terminal
+    Run Command    /usr/bin/x-terminal-emulator
```

Comment:
Can drop `/usr/bin`  and maybe call `xdg-terminal`, since we are not tergetting noble anyways here

Reply:
Even though we are not targeting Noble (yet, at least), I think it's better to keep the tests as release-independent as possible. If we change that, we would need to have multiple versions of the resource files (and although this might eventually become the case in the future, we should avoid it for now)

Reply:
As you prefer, the `/us/bin/` drop still stands :)